app/controllers/merchant/dashboard_controller.rb

```diff
@@ -2,6 +2,7 @@ class Merchant::DashboardController < ApplicationController
   before_action :require_merchant
 
   def index
+    @merchant = current_user.merchant
```

Comment:
If I understand correctly, this returns the merchant linked to the user. This should help with user story 31, but I am still not sure how I will get all the order from this specific merchant. Maybe something like `merchant.item_orders` since the merchant has many items and item_orders belong to merchants.

Reply:
Maybe you can create a method in the Merchant model that grabs all a merchant's orders...

Something where you pass in an `order_id` to the method itself (e.g., `def merchant_item_orders(order_id) end`), and the method would be like
`item_orders.joins` the item WHERE the `order_id` matches the one passed in the argument of the method, and the `items.merchant_id` matches the merchant you want...

Reply:
I agree with Kiera